frontend/src/pages/Policy/policy.jsx

```diff
@@ -5,8 +5,218 @@ import './policy.css';
+                    їхній вибір та взаємодію з різними елементами Сайту.`,
+
+                    `Користувач має можливість керувати використанням файлів 
+                    "cookies" у налаштуваннях свого веб-браузера. Відключення 
+                    файлів "cookies" може бути здійснено користувачем у будь-який 
+                    момент. Проте важливо зауважити, що відключення "cookies" 
+                    може призвести до обмеження функціональності Сайту, а також 
+                    до втрати певних персоналізованих налаштувань та зручностей, 
+                    які вони забезпечують.`,
+                ],
+            },
+            {
+                // Paragraph 5
+                title: 'Зміни у Політиці конфіденційності',
+                text: [
+                    `Адміністрація Сайту залишає за собою право вносити зміни 
+                    у цю Політику конфіденційності за власною ініціативою або 
+                    відповідно до змін у законодавстві або вимогах безпеки даних. 
+                    Зміни можуть включати в себе додаток нових правил, виправлення 
+                    помилок або уточнення існуючих положень. Нова версія Політики 
+                    конфіденційності набуває чинності з моменту її публікації 
+                    на Сайті. Інформація про внесені зміни може бути доступна 
+                    шляхом оголошень на Сайті або шляхом інших засобів 
+                    повідомлення користувачів.`,
+
+                    `Користувачі мають зобов'язання періодично переглядати цю 
+                    Політику конфіденційності на Сайті для ознайомлення з будь-якими 
+                    змінами або оновленнями. При продовженні використання Сайту 
+                    після внесення змін користувачі автоматично погоджуються з 
+                    оновленою версією Політики. Якщо користувач не погоджується 
+                    з будь-якою зміною у цій Політиці, йому рекомендується 
+                    припинити використання Сайту.`,
+                ],
+            },
+        ],
+    };
+
+    /**
+     * Render text block
+     * @param textBlock - text block (string or list)
+     * @param index - index of the text block
+     * @param paraIndex - index of the paragraph
+     * @return {JSX.Element|null}
+     */
+    const renderText = (textBlock, index, paraIndex) => {
+        if (typeof textBlock === "string") {
+            return (
+                <p
+                    key={index}
+                    aria-label={`Пункт ${paraIndex + 1}.${index + 1}`}
+                >
+                    {(paraIndex + 1) + "." + (index + 1) + ". " + textBlock}
+                </p>
+            );
+        }
+
+        if (textBlock.type === "list") {
+            return (
+                <ul
+                    key={index}
+                    className={'ul'}
+                    role="list"
+                    aria-label={`Список у пункті ${paraIndex + 1}`}
+                >
```

Comment:
Hardcoded class names like 'ul' and 'li' may not match BEM or project CSS conventions. Use more descriptive, unique class names based on project conventions.
```
<ul className="policy__list" ...>
    <li className="policy__list-item" ...>
        {item}
    </li>
</ul>
```